### DISCUSSION TITLE
Press and hold to change screens quicker

### DISCUSSION THREAD

**Discussion by Lolagatorade** [meshtastic/firmware#4420] [Ideas]
### Platform

ESP32

### Description

On heltech v3 it takes a while to click and at times the button doesn’t register without a firm click therefore a press and hold would be much more effective for changing screens

Especially considering…. Environmental measurements and device info are opposed on complete opposite side of the screen sandwiched between node data

For ease to the user press and hold for rapid scroll / carousel would be beneficial to the UX

**Comment by Dooomenik** [comment]
With press and hold you could eventually miss the screen you wanted and have to do it again. Maybe it would be a good solution to use press and hold to switch "categories" for example messages, nodes, device info. And than navigate inside these categories with simple clicks. With this feature you could also better navigate to last messages. With this and more than just the last message from a node (last 3 for example, depends on display size) you can better use meshtastic nodes with a rotary encoder or even a keyboard in emergency situations if your mobile is dead.
